### PR TITLE
Add ratings test to dev regression

### PR DESCRIPTION
**Pull request by AlexandraMoga**
This PR includes:
- variables necessary for running collections, users and ratings tests on AMO -dev
- configs for the above mentioned suites
- a few minor changes to some of the tests

### CODE REVIEW

File: tests/frontend/test_collections.py [hunk 1]
```diff
@@ -495,21 +497,14 @@ def test_create_collection_with_empty_custom_url(selenium, base_url):
...
     collections.create.save_collection()
     collections.collection_detail.wait_for_details_to_load()
-    # go back and try to create another collection with different name, but same URL
-    collections.driver.get('https://addons.allizom.org/en-US/firefox/collections/')
-    collections.click_create_collection()
-    collections.create.set_name('def')
-    collections.create.slug_label_element.clear()
-    collections.create.set_slug('abc')
-    collections.create.save_collection()
     assert variables['collection_reused_url_warning'] in collections.create.warning_text
```
Comment (AlexandraMoga) [inline tests/frontend/test_collections.py:508]: This test was having too many unnecessary steps. I've made a more simple version for it which achieves the same goal.

File: tests/frontend/test_collections.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import time
```
Comment (jrbenny35) [inline tests/frontend/test_collections.py:1]: I don't think you used this

**Review by jrbenny35 — CHANGES_REQUESTED** [review]
Just one thing

File: tests/frontend/test_collections.py (hunk 2 above)
Comment (AlexandraMoga) [reply tests/frontend/test_collections.py:1]: Ugh, I forgot to clean up my imports. I'll remove it.

Comment (jrbenny35) [reply tests/frontend/test_collections.py]: We could add flake8 to help with that!

**Review by jrbenny35 — APPROVED** [review]

**Pull request merged**